extensions/huawei/GLSL_HUAWEI_subpass_shading.txt

```diff
@@ -0,0 +1,147 @@
+Name
+
+    HUAWEI_subpass_shading
+
+Name Strings
+
+    GL_HUAWEI_subpass_shading
+
+Contact
+
+    Hueilong Wang (wyvernathuawei), HUAWEI
+
+Contributors
+
+    Hueilong Wang, HUAWEI
+
+Status
+
+    Draft
+
+Version
+
+    Last Modified Date: 2021-04-26
+    Revision: 1
+
+Dependencies
+
+    This extension can be applied to OpenGL GLSL versions 4.60
+    (#version 460) and higher.
+
+    This extension can be applied to OpenGL ES ESSL versions 3.20
+    (#version 320) and higher.
+    
+    This extension is written against revision 7 of the OpenGL Shading Language
+    version 4.60, dated July 10, 2019.
+
+    This extension is written against revision 6 of the OpenGL ES ESSL
```

Comment:
I don't think you need to include the actual changes twice, you can just say that the extension can be applied to ES, but that it's written against the desktop version. If there are any significant differences between desktop and ES then you can note them at the end in a section of "Interactions with Open GL ES". Sorry to have misled you on this.